tests/authServer.test.js

```diff
@@ -0,0 +1,44 @@
+const dotenv = require('dotenv');
```

Comment:
You do this on each test... maybe you should do it instead in a jest setup file.